DeviceAdministration/Infrastructure.UnitTests/Simulator.WebJob/JsonSerializeTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.Azure.Devices.Applications.RemoteMonitoring.Simulator.WebJob.SimulatorCore.Serialization;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.UnitTests.Simulator.WebJob
+{
+    class JsonSerializeTests
+    {
+        private JsonSerialize serializer;
```

Comment:
actually, on second thought, it is only used in two places but the plumbing of custom serialization through the transport might be worth keeping it for the future